frame/evm/precompile/modexp/src/lib.rs

```diff
@@ -20,12 +20,71 @@
 extern crate alloc;
 
 use alloc::vec::Vec;
-use fp_evm::LinearCostPrecompile;
-use evm::{ExitSucceed, ExitError};
+use fp_evm::Precompile;
+use evm::{ExitSucceed, ExitError, Context};
 use num::{BigUint, Zero, One, ToPrimitive, FromPrimitive};
 
+use core::ops::BitAnd;
+
 pub struct Modexp;
 
+const MIN_GAS_COST: u64 = 200;
+
+// TODO: is there something in Substrate I can use?
```

Comment:
Yes! `core::cmp::max`

Reply:
I thought I ran into `no_std` issues when I tried that -- seems fine now. Fixed in 56562ed